evm/utils.js

```diff
@@ -751,6 +751,8 @@ const mainProcessor = async (options, processCommand, save = true, catchErr = fa
         return;
     }
 
+    const chainsSnapshot = JSON.parse(JSON.stringify(config.chains));
```

Comment:
```suggestion
    const chainsSnapshot = deeepcopy(config.chains);
```
create a common util with a more meaningful name?